### PR TITLE
test: Add a bad error msg to track list

### PR DESCRIPTION
**Pull request by khoa165**
# Changes
- Add a test for a bad error message when query sort decreasing order but without square bracket

## Type
- Dev 

## Issue
https://github.com/PRQL/prql/issues/2282

### CODE REVIEW

File: prql-compiler/src/tests/test_bad_error_messages.rs [hunk 1]
```diff
@@ -71,6 +71,20 @@ fn test_bad_error_messages() {
        │     Note: Type `relation` expands to `[tuple_of_scalars]`
     ───╯
     "###);
+
+    // It's better if we can tell them to put in [] bracketß
```
Comment (max-sixty) [inline prql-compiler/src/tests/test_bad_error_messages.rs]:
```suggestion
    // It's better if we can tell them to put in {} braces
```

**Review by max-sixty — APPROVED** [review]
Perfect first PR. Great example of a terrible error message...

File: prql-compiler/src/tests/test_bad_error_messages.rs (hunk 1 above)
Comment (khoa165) [reply prql-compiler/src/tests/test_bad_error_messages.rs]: Is the book outdated too? Cuz I see square bracket being used there (screenshot below)
Screenshot 2023-06-20 at 1 16 44 AM

Comment (max-sixty) [reply prql-compiler/src/tests/test_bad_error_messages.rs]: Yes, all the docs are on 0.8.1; then `main` has the new syntax

Comment (max-sixty) [reply prql-compiler/src/tests/test_bad_error_messages.rs]: `task run-book` to see the current book locally...

**Pull request merged**